### ISSUE TITLE
Recipe package should be "lightweight"

### ISSUE THREAD

**Issue by dlfivefifty** [JuliaPlots/AbstractPlotting.jl#53]
To add a recipe for Makie requires this package, but the number of dependencies is very large. One of the nice things about RecipesBase.jl is that it is really light.

Is there a plan to make the basic recipes package as lightweight?

**Comment by SimonDanisch** [comment]
Yeah, there is a plan, but it doesn't have high priority untill we figured out a stable Recipe interface - which isn't too far away thanks to @piever :)
